release/models/network-instance/openconfig-evpn.yang

```diff
@@ -12,6 +12,7 @@ module openconfig-evpn {
   import openconfig-inet-types { prefix oc-inet; }
   import openconfig-interfaces { prefix oc-if; }
   import openconfig-bgp-types { prefix oc-bgp-types; }
+  import ietf-yang-types { prefix yang; }
```

Comment:
We have transitioned away from this dependency - it looks like you can use `oc-types:timeticks64` here.

Reply:
I can change this to use `oc-types:timeticks64` but we are a bit unclear on the usage based on the description.

```yang
typedef timeticks64 {
    type uint64;
    units "nanoseconds";
    description
     "The timeticks64 represents the time, modulo 2^64 in
     nanoseconds between two epochs. The leaf using this
     type must define the epochs that tests are relative to.";
  } 
```

Is there a good use case or example where this type is used?  We don't understand what `The leaf using this
     type must define the epochs that tests are relative to.` means...

Reply:
Changes to use `oc-types:timeticks64`